libs/libdict/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdict
+PKG_RELEASE=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/fmela/libdict.git
+PKG_SOURCE_DATE:=2018-05-20
+PKG_SOURCE_VERSION:=95f6d27f277932272578b2554104e1e4414cfaac
+PKG_MIRROR_HASH:=6231b7d874807b333f8b1f2821f450a554c002b07d2248d04b9cfed40e2adec4
+
+PKG_LICENSE:=BSD
```

Comment:
This should look like:

PKG_MAINTAINER:=Rosy Song <rosysong@rosinson.com>
PKG_LICENSE:=BSD-2-Clause
PKG_LICENSE_FILES:=LICENSE

PKG_INSTALL:=1